packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -94,6 +94,10 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ISemver {
     /// @notice Mapping of withdrawal hashes to addresses that have submitted a proof for the withdrawal.
     mapping(bytes32 => address[]) public proofSubmitters;
 
+    /// @custom:spacer _balance (custom gas token)
+    /// @notice Spacer for backwards compatibility.
```

Comment:
nit, but technically more precise? 
```suggestion
    /// @notice Spacer for forwards compatibility.
```

Reply:
You are correct this is more precise but tbh I don't want to deal with going thru semver lock + flakey CI again

Reply:
We'll need a semver bump anyways due to the storage layout change. Sorry sir, know it's painful

Reply:
I have addressed this comment